src/yunohost/app.py

```diff
@@ -1470,13 +1458,12 @@ def _sanitized_absolute_url(perm_url):
 
 
 def app_change_label(app, new_label):
+    from permission import user_permission_update
     installed = _is_installed(app)
     if not installed:
         raise YunohostError('app_not_installed', app=app, all_apps=_get_all_installed_apps_id())
-
-    app_setting(app, "label", value=new_label)
-
-    app_ssowatconf()
+    logger.warning(m18n.n('app_label_deprecated'))
```

Comment:
Re-reading this, I don't know what to think about this deprecation ... on one hand, I understand the will to have only one consistent way to manage all aspects of permissions ... on the other hand, 'yunohost user permission update' ain't trivial to use and weird when what you want is just to change the label of an app (anyway the main way to do this should be using the webadmin and that should be transparent for the admin, but zbleh)